docs/reference/src/main/asciidoc/configure.asciidoc

```diff
@@ -1,6 +1,7 @@
 [[configure]]
 == Configuration
 
+[[weld-configuration]]
```

Comment:
Cool, just what I was looking for. I copied this solution from other part of the doc since I wasn't sure how this works. Thanks!